README.md

```diff
@@ -88,7 +88,7 @@ To create a sample:
 To build and run your sample:
 
 
-1. Go to the sample folder and Build to check for errors.
+1. Go to the sample folder and Build to check for errors:
```

Comment:
- "Build" here should be "build" 
- If line 92 ends with a colon, so should line 96 (replace "Run!" with "Run your sample:".